RecoLocalTracker/SiPixelRecHits/src/SiPixelTemplateSplit.cc

```diff
@@ -47,7 +47,7 @@
 #include "FWCore/MessageLogger/interface/MessageLogger.h"
 #define LOGERROR(x) edm::LogError(x)
 #define LOGDEBUG(x) LogDebug(x)
-static int theVerboseLevel = 2;
+static constexpr int theVerboseLevel = 2;
```

Comment:
Given all the stuff I have on my plate and that we use ```static``` much more than unnamed namespaces I'd prefer to keep it as it is.